ci/cirrus_general_ci.yml

```diff
@@ -0,0 +1,87 @@
+# Regular CI for testing linux_aarch64 natively
```

Comment:
How come this has crept in? It was removed in  #22446?

Reply:
You mean this file? I will remove it.